static/app/views/alerts/rules/details/metricHistory.tsx

```diff
@@ -0,0 +1,244 @@
+    return t('Critical');
+  }
+
+  // Otherwise, activity type is not status change
+  return '';
+}
+
+type Props = {
+  organization: Organization;
+  incidents?: Incident[];
+};
+
+function MetricHistory({organization, incidents}: Props) {
+  const renderActivity = incident => {
+    const activities = incident!.activities!.filter(
+      activity => activity.type === IncidentActivityType.STATUS_CHANGE
+    );
+    const criticalActivity = activities.filter(
+      activity => activity.value === `${IncidentStatus.CRITICAL}`
+    );
+    const warningActivity = activities.filter(
+      activity => activity.value === `${IncidentStatus.WARNING}`
+    );
+
+    const triggeredActivity = !!criticalActivity.length
+      ? criticalActivity[0]
+      : warningActivity[0];
+    const currentTrigger = getTriggerName(triggeredActivity.value);
+
+    const nextActivity = activities.find(
+      ({previousValue}) => previousValue === triggeredActivity.value
+    );
+
+    const activityDuration = (
+      nextActivity ? moment(nextActivity.dateCreated) : moment()
+    ).diff(moment(triggeredActivity.dateCreated), 'milliseconds');
+
+    const threshold =
+      activityDuration !== null &&
+      tct('[duration]', {
+        duration: <Duration abbreviation seconds={activityDuration / 1000} />,
+      });
+
+    const warningThreshold = incident.alertRule.triggers
+      .filter(trigger => trigger.label === 'warning')
+      .map(trig => trig.alertThreshold);
+    const criticalThreshold = incident.alertRule.triggers
+      .filter(trigger => trigger.label === 'critical')
+      .map(trig => trig.alertThreshold);
+
+    return (
+      <ErrorBoundary>
+        <Title data-test-id="alert-title">
+          <StatusLevel level={currentTrigger}>
+            <Tooltip title={tct('Status: [level]', {level: currentTrigger})}>
+              <span />
+            </Tooltip>
+          </StatusLevel>
+          <Link
+            to={{
+              pathname: alertDetailsLink(organization, incident),
+              query: {alert: incident.identifier},
+            }}
+          >
+            {tct('#[id]', {id: incident.identifier})}
+          </Link>
+        </Title>
+        <Cell>
+          {tct('[title] [selector] [threshold]', {
+            title:
+              AlertWizardAlertNames[getAlertTypeFromAggregateDataset(incident.alertRule)],
+            selector:
+              incident.alertRule.thresholdType === AlertRuleThresholdType.ABOVE
+                ? 'above'
+                : 'below',
+            threshold:
+              currentTrigger === 'Warning' ? warningThreshold : criticalThreshold,
+          })}
+        </Cell>
+        <Cell>{threshold}</Cell>
+        <Cell>
+          <IconWrapper>
+            <Tooltip title={t('People who have viewed this')} skipWrapper>
+              <StyledIconShow size="xs" color="gray200" />
+            </Tooltip>
+            {incident.seenBy.length}
+          </IconWrapper>
```

Comment:
In other parts of the UI we show an avatar list of the people who have viewed an issue or contributed to a release, are we planning to do that here in the future?

Reply:
I think we had that at some point and its kinda been rotating